tests/console/dns_srv.pm

```diff
@@ -33,7 +33,7 @@ sub run {
     # verify dns server responds to anything
     my $e = script_run "host localhost localhost";
     if ($e) {
-        record_soft_failure 'Bug 1064438: "bind" cannot resolve localhost' if check_var('ARCH', 's390x');
```

Comment:
@SergioAtSUSE I think you added this. Please always use `boo#<id>` or `bsc#<id>`

Reply:
@okurz please explain why, I guess there is a reason..

Reply:
Well, the reason is that openQA itself replaces `boo#<id>` or `bsc#<id>` by a proper URL as well as it does in comments. E.g. when you just copy-paste a full bug URL it will be replaced by the according html `<a href=…` with the text `bsc#<id>`. So if we stick to the convention of writing bug references following this usual format then also the soft-fail references are clickable URLs.

This has been mentioned to me by behlert.

Reply:
Thanks, is it also required by this?
```
This is an autogenerated message for openQA integration by the openqa_review
script:

This bug is still referenced in a failing openQA test: clone_system
https://openqa.opensuse.org/tests/680847
```